functions/src/strings.js

```diff
@@ -94,6 +94,13 @@ module.exports = {
       suggestions: ['reset'],
     },
 
+    /**
+     * we got 4xx, 5xx response from a server
+     */
+    httpRequestError: {
+      speech: 'We experience some technical problems on IA server, please try again later or try something else.',
```

Comment:
Change this line to `We are currently experiencing some technical difficulties on the Archive server. Please try again later or try saying something else.`.